test/ttmlir/Silicon/TTNN/sharded/mnist_sharding_tiled.mlir

```diff
@@ -1,18 +1,18 @@
-// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path% enable-optimizer=true memory-layout-analysis-enabled=true override-output-layout=matmul_1=1x8:l1:width_sharded,add_2=1x8:l1:width_sharded,relu_3=1x8:l1:width_sharded,matmul_5=1x1:l1:width_sharded,add_6=1x1:l1:width_sharded,softmax_7=1x1:l1:width_sharded" %s > %t.mlir
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path% enable-optimizer=true memory-layout-analysis-enabled=true override-output-layout=matmul_1=1x8:l1:width_sharded:row_major:f32,add_2=1x8:l1:width_sharded:row_major:f32,relu_3=1x8:l1:width_sharded:row_major:f32,matmul_5=1x1:l1:width_sharded:row_major:f32,add_6=1x1:l1:width_sharded:row_major:f32,softmax_7=1x1:l1:width_sharded:row_major:f32" %s > %t.mlir
```

Comment:
Is each segment optional or mandatory?

Reply:
Everything is mandatory now. Making it optional requires a larger change in how we handle analysis overrides so it will be part of a separate PR.